src/db/types.ts

```diff
@@ -16,7 +16,11 @@ export interface DBGuildChannel extends BaseModel {
 }
 
 export interface DBGuildMember extends BaseModel {
+  /** @deprecated use aboutText instead */
```

Comment:
I know it's more work, but could we potentially migrate these (via a script) instead of just deprecating them? Using this pattern opens us up to having a lot of deprecated fields in the future as schema changes are continually made.

Reply:
I agree. It was this change that led me to making a `rusty-dev` project so that when we run tests like this it would not pollute the main db.

tbh I have no idea how one would do this sort of migration haha. I'm open to suggestions! That and I probably should find a way to scan the current roles in the server to populate this.

Reply:
I'm sure this could be handled with either a JS script or SQL script to pull the values, update them, and then update the stored information.